src/modules/robot/arm_solutions/RotatableDeltaSolution.cpp

```diff
@@ -0,0 +1,242 @@
+#include "RotatableDeltaSolution.h"
+#include "checksumm.h"
+#include "ConfigValue.h"
+#include "ConfigCache.h"
+#include "libs/Kernel.h"
+#include "libs/nuts_bolts.h"
+#include "libs/Config.h"
+#include "libs/utils.h"
+#include "StreamOutputPool.h"
+#include <fastmath.h>
+
+#define delta_e_checksum       			CHECKSUM("delta_e_checksum")
+#define delta_f_checksum   	    		CHECKSUM("delta_f_checksum")
+#define delta_re_checksum       		CHECKSUM("delta_re_checksum")
+#define delta_rf_checksum       		CHECKSUM("delta_rf_checksum")
+#define delta_z_offset_checksum 		CHECKSUM("delta_z_offset_checksum")
+
+#define delta_ee_offs_checksum 		  	CHECKSUM("delta_ee_offs_checksum")
+#define tool_offset_checksum			CHECKSUM("tool_offset_checksum")
+
+#ifndef alpha_steps_per_mm_checksum
```

Comment:
please remove these three as we do not like #ifdefs in the code is makes it harder to read and these appear to not be needed anywhere